tests/test_suites/LongSystemTests/test_build_while_changing_master.sh

```diff
@@ -55,9 +55,9 @@ master_kill_loop() {
 master_kill_loop &
 
 cd "${info[mount0]}"
-assert_success git clone https://github.com/saunafs/saunafs.git
+assert_success git clone https://github.com/leil-io/saunafs.git
 saunafs setgoal -r 2 saunafs
 mkdir saunafs/build
 cd saunafs/build
-assert_success cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../install
+assert_success cmake .. -G 'Unix Makefiles' -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=../install
```

Comment:
is it for making sure that under Windows we have still Unix make-files ? 

... what this `-G 'Unix Makefiles'` is for ?

Reply:
The cmake tool has different types of generator (Unix Makefiles, Ninja, Visual Studio, Eclipse, CodeBlocks, kDevelop3, etc). The -G 'Unix Makefiles' instruct cmake to use the Unix/Linux Makefile generator because we are building the code using make command after that line.